stable/mission-control/README.md

```diff
@@ -250,12 +250,71 @@ View specific log
 kubectl logs -n <NAMESPACE> <POD_NAME> -c <LOG_CONTAINER_NAME>
 ```
 
+### Custom sidecar containers
+There are cases where an extra sidecar container is needed. For example monitoring agents or log collection.
+
+For this, there is a section for writing a custom sidecar container in the [values.yaml](values.yaml). By default it's commented out
+```yaml
+common:
+  ## Add custom sidecar containers
+  customSidecarContainers: |
+    ## Sidecar containers template goes here ##
+```
+
+### Establishing TLS and Adding certificates
+Create trust between the nodes by copying the ca.crt from the Artifactory server under $JFROG_HOME/artifactory/var/etc/access/keys to of the nodes you would like to set trust with under $JFROG_HOME/<product>/var/etc/security/keys/trusted. For more details, Please refer [here](https://www.jfrog.com/confluence/display/JFROG/Managing+TLS+Certificates).
+
+
+To add this certificate to missioncontrol, Create a configmaps.yaml file with the following content:
+
+```yaml
+common:
+  configMaps: |
+    ca.crt: |
+      -----BEGIN CERTIFICATE-----
+        <certificate content>
+      -----END CERTIFICATE-----
+  customVolumeMounts: |
+    - name: mission-control-configmaps
+      mountPath: /tmp/ca.crt
+      subPath: ca.crt
+missionControl:
+  preStartCommand: "mkdir -p {{ .Values.missionControl.persistence.mountPath }}/etc/security/keys/trusted && cp -fv /tmp/ca.crt {{ .Values.missionControl.persistence.mountPath }}/etc/security/keys/trusted/ca.crt"
+router:
+  tlsEnabled: true  
+```
+
+and use it with you helm install/upgrade:
+```bash
+helm upgrade --install xray -f configmaps.yaml --namespace xray center/jfrog/xray
+```
+
+This will, in turn:
+* Create a configMap with the files you specified above
+* Create a volume pointing to the configMap with the name `mission-control-configmaps`
+* Mount said configMap onto `/tmp` using a `customVolumeMounts`
+* Using preStartCommand copy the `ca.crt` file to xray trusted keys folder `/etc/security/keys/trusted/ca.crt`
+* `router.tlsEnabled` is set to true to add HTTPS scheme in liveness and readiness probes.
+
+### Custom volumes
+
+If you need to use a custom volume in a custom init or sidecar container, you can use this option.
```

Comment:
Is this only for init or sidecar containers?

Reply:
Updated